extensions/bit-manipulation-1/README.md

```diff
@@ -16,34 +16,31 @@ These instructions are in the expanded calculation opcode section of instruction
 
 | `C CCCC CCCC` | NAME      | Operation                                              | Flags | Comment     |
 |---------------|-----------|--------------------------------------------------------|-------|-------------|
-| `0 0000 1000` | `RCL`     | <code>C:A ← (C:A << B) &#124; (C:A >> -B)</code>       | `ZNC` | (1) (2) (3) |
-| `0 0000 1001` | `RCR`     | <code>A:C ← (A:C >> B) &#124; (A:C << -B)</code>       | `ZNC` | (1) (2) (3) |
+| `0 0000 1000` | `RCL`     | <code>C:A ← (C:B << 1) &#124; (C:B >> -1)</code>       | `ZNC` | (1) (2) (3) |
+| `0 0000 1001` | `RCR`     | <code>A:C ← (B:C >> 1) &#124; (B:C << -1)</code>       | `ZNC` | (1) (2) (3) |
```

Comment:
I think now that it's fixed shift, it would be better to write `B ← (B << 1) | C; C = msb(B)` or something like that.

Reply:
The `C:B` notation covers setting the `C` flag (and is explained again in a comment) so I think that that's OK. However using `| C` for the bottom bit is clearly better. Making an update shortly.

Reply:
Yes, but IMO it would better to not require the comment explaining a tbh. slightly arcane syntax for a situations where it isn't required. Saying "`msb(B)` refers to the most significant bit of `B` for the correct width" is IMO clearer and easier to understand.